src/app.rb

```diff
@@ -0,0 +1,149 @@
+require_relative './musicalbum'
+require_relative './gaming_methods'
+require_relative './book_methods'
+require_relative './label'
+require 'json'
+
+class App
+  attr_accessor :album_list, :genre_list, :game_list, :author_list
+
+  def initialize()
+    @music_list = []
+    @genre_list = []
+    @game_list = []
+    @author_list = []
+    @book_list = []
+    @label_list = []
+  end
+
+  def music_display
+    list_music
+  end
+
+  def genre_display
+    list_genres
+  end
```

Comment:
- Please note that I get some errors when I try to list genres. Kindly fix it. 👍🏽
![Screenshot from 2023-01-27 21-18-15](https://user-images.githubusercontent.com/44624138/215190617-bd641e1e-e1a9-4e35-8f10-85ce8c7ce5e7.png)